.github/workflows/static-analysis.yml

```
@@ -0,0 +1,85 @@
+
+on:
+  pull_request: ~
+  push: ~
+
+name: Static analysis
+
+jobs:
+  phpstan:
+    name: PHPStan
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php-version: [7.4]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
+        with:
+          coverage: none
+          extensions: intl
+          php-version: ${{ matrix.php-version }}
+
+      - name: Determine composer cache directory
+        id: determine-composer-cache-directory
+        run: echo "::set-output name=directory::$(composer config cache-dir)"
+
+      - name: Cache dependencies installed with composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.determine-composer-cache-directory.outputs.directory }}
+          key: php-${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.json') }}
```

```suggestion
          key: php-${{ matrix.php-version }}-composer-${{ hashFiles('composer.*') }}
```
There's no need to consider `composer.*` files in other than root directory.